app/Http/Controllers/Api/SessionController.php

```diff
@@ -59,7 +70,15 @@ public function update(Request $request)
             $request->get('session') != null ? $session->session = $request->get('session') : null;
             $request->get('is_current') != null ? $session->is_current = $request->get('is_current') : null;
 
-            if($session->save()){
+            if($session->save()){                
+
+                $is_current = $request->get('is_current');
```

Comment:
Isn't this repetition? I'm thinking since we have setSession() method.

Reply:
resolved, thanks 👍